creator-node/src/services/stateMachineManager/stateMonitoring/index.js

```diff
@@ -102,14 +101,6 @@ class StateMonitoringManager {
       prometheusRegistry
     })
 
-    // Clear any old state if redis was running but the rest of the server restarted
-    if (clusterUtils.isThisWorkerInit()) {
```

Comment:
why remove these?
also the isThisWorkerInit() checks below

Reply:
I moved it into `makeQueue()` (with the isThisWorkerInit() check) because I needed it to clear before the worker is created (so it doesn't try to clear while processing jobs, which is one cause of the missing lock error)